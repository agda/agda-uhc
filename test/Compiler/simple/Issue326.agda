
module Issue326 where

open import Common.Prelude

postulate
  QName : Set

{-# BUILTIN QNAME QName #-}

primitive primQNameEquality : QName → QName → Bool

main : IO Unit
main = printBool (primQNameEquality (quote Unit) (quote IO))
