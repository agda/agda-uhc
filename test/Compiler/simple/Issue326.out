EXECUTED_PROGRAM

ret > ExitSuccess
out > false
